cfgov/v1/models/home_page.py

```diff
@@ -273,6 +199,9 @@ class HomePage(CFGOVPage):
         ObjectList(content_panels, heading='General Content'),
         ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
         ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+        ObjectList([
+            InlinePanel('carousel', min_num=4, max_num=4, label="Carousel"),
```

Comment:
Naming this `carousel_items` might make the call in the template (`{{ carousel.render( page.carousel.all() ) }}` :arrow_right: `{{ carousel.render( page.carousel_items.all() ) }}`) a bit more clear. The former sorta implies that you can have multiple carousels.

Reply:
Originally I did try to name this carousel_items. Unfortunately, that conflicts with [this similar reference](https://github.com/wagtail/wagtail/blob/b9c470df37ee971ad6d60827e631e36897b168e6/wagtail/tests/testapp/models.py#L194) in the Wagtail test app, which caused our tests to break since we include that app when we run them. Looking at this again, though, I might be able to get around that by changing the ParentalKey from Page to CFGOVPage. I'll give that a try.

Reply:
This change worked. By adding the `carousel_items` reference to CFGOVPage instead of Page, it no longer conflicts with the Wagtail test app.